# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/text/reftest.list
# Commit: e24ed89246a4
# Full Hash: e24ed89246a4a6c6b4a567ec412f0e3295b4570d
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-04-26 09:46:09
# Regressor Bug: 1766017
# File Overlap Count: 1
# Description:
#   Bug 1766017 - Do a better job of clipping off-screen surface allocation sizes r=gfx-reviewers,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D144534
# ==============================================================================

diff -r cdc42b7822f1 -r e24ed89246a4 gfx/wr/wrench/reftests/text/reftest.list
--- a/gfx/wr/wrench/reftests/text/reftest.list	Mon Apr 25 22:28:35 2022 +0000
+++ b/gfx/wr/wrench/reftests/text/reftest.list	Mon Apr 25 23:40:40 2022 +0000
@@ -81,7 +81,7 @@
 # most pixels are off by a small amount, but a few pixels on the edge vary by a lot, pushing up the fuzzy max-diff;
 # the main goal of the test is that everything is in the same place, at the same scale, clipped the same way,
 # despite 4x on-the-fly scale change.
-skip_on(android) fuzzy(110,20200) == raster_root_C_8192.yaml raster_root_C_ref.yaml
+skip_on(android) fuzzy(120,20800) == raster_root_C_8192.yaml raster_root_C_ref.yaml
 == subpx-bg-mask.yaml subpx-bg-mask-ref.yaml
 platform(linux,mac) == rotate-snap-clip.yaml rotate-snap-clip-ref.yaml
 platform(linux,mac) == rotate-snap-filter.yaml rotate-snap-filter-ref.yaml